phoenicis-containers/src/main/java/org/phoenicis/containers/wine/WinePrefixContainerController.java

```diff
@@ -150,6 +150,21 @@ public void deletePrefix(WinePrefixContainerDTO winePrefix, Consumer<Exception>
                 });
     }
 
+    public void createShortcut(WinePrefixContainerDTO winePrefix, String name, String executable, Runnable doneCallback,
```

Comment:
Please add some JavaDoc containing what these variables mean/contain, especially `name` and `executable`.